descqa/shear_test.py

```diff
@@ -206,26 +211,26 @@ def get_catalog_data(gc, quantities, filters=None):
 
     # define theory from within this class
 
-    def post_process_plot(self, ax):
+    def post_process_plot(self, ax, fig):
         '''
         Post-processing routines on plot
         '''
-        zmeans = np.linspace(self.zlo, self.zhi, self.ntomo+2)[1:-1]
+        #zmeans = np.linspace(self.zlo, self.zhi, self.ntomo+2)[1:-1]
 
         # vmin and vmax are very rough DES-like limits (maximum and minimum scales)
         for i in range(self.ntomo):
             for ax_this, vmin, vmax in zip(ax[:, i], (2.5, 35), (200, 200)):
                 ax_this.set_xscale('log')
                 ax_this.axvline(vmin, ls='--', c='k')
                 ax_this.axvline(vmax, ls='--', c='k')
-            ax[-1][i].set_xlabel(r'$\theta \; {\rm (arcmin)}$')
-            ax[0][i].set_title('z = '+str(zmeans[i]))
-            ax[0][i].legend()
-            ax[-1][i].legend()
-        ax[0][0].set_ylabel(r'$\chi_{{{}}} \; (10^{{-6}})$'.format('+'))
-        ax[-1][0].set_ylabel(r'$\chi_{{{}}} \; (10^{{-6}})$'.format('-'))
-
+            ax[-1][i].set_xlabel(r'$\theta \; {\rm (arcmin)}$', size=self.axsize)
+            ax[0][i].set_title('z = {:.2f}'.format(self.zmeans[i]), size=self.title_size)
+            ax[0][i].legend(fontsize=self.legend_size)
+            ax[-1][i].legend(fontsize=self.legend_size)
+        ax[0][0].set_ylabel(r'$\chi_{{{}}} \; (10^{{-6}})$'.format('+'), size=self.axsize)
+        ax[-1][0].set_ylabel(r'$\chi_{{{}}} \; (10^{{-6}})$'.format('-'), size=self.axsize)
 
+        fig.subplots_adjust(hspace=0)
```

Comment:
To maintain compatibility with other calls of this function..
```suggestion
        if fig is not None:
            fig.subplots_adjust(hspace=0)
```

Reply:
fig should have been in the call to the function.  I just forgot it.